lib/tinybucket/model/profile.rb

```diff
@@ -30,6 +30,13 @@ class Profile < Base
         :username, :kind, :website, :display_name,
         :links, :created_on, :location, :type, :uuid
 
+      # Get this user's email addresses
+      #
+      # @return [Tinybucket::Resource::Emails]
+      def emails
+        Tinybucket::Resource::Emails.new
+      end
+
```

Comment:
This class should have `email` method which is described [here](https://github.com/hirakiuc/tinybucket/pull/105/files#r98321602).